lib/SmlMessageBody.js

```diff
@@ -23,139 +23,109 @@ var SmlSetProcParameterRequest = require('./messages/SmlSetProcParameterRequest'
 var SmlSetProcParameterResponse = require('./messages/SmlSetProcParameterResponse');
 
 function SmlMessageBody(messageTag){
-	
+
 	switch(messageTag) {
 		case Constants.PUBLIC_OPEN_REQUEST:
 			return new SmlPublicOpenRequest();
-			break;
```

Comment:
removed because the return exists the function at all, so the break is never reached. Because of this with a return in there the break is somewhat "implicit"